add RACK and QUIC loss detection (#81)
NTAP/rfc8312bis: draft-ietf-tcpm-rfc8312bis.md

functions, CUBIC uses both the concave and convex profiles of a cubic
function for window growth.

After a window reduction in response to a congestion event detected
by duplicate ACKs, Explicit Congestion Notification-Echo (ECN-Echo,

> ECE) ACKs {{!RFC3168}}, TCP RACK {{!RFC8985}} or QUIC loss detection,

Reviewer: ```suggestion
ECE) ACKs {{!RFC3168}}, TCP RACK {{!RFC8985}} or QUIC loss detection {{!RFC9002}},
```

Author: I had this in the first commit and I had to remove as it failed to compile. Does 9002 have a valid reference?

Reviewer: It should? Let me try locally.

> CUBIC remembers the congestion window size where it received the congestion

Reviewer: ```suggestion
CUBIC remembers the congestion window size at which it received the congestion
```

does not make any changes to the TCP Fast Recovery and Fast Retransmit
algorithms {{!RFC6582}}{{!RFC6675}}.

> During congestion avoidance after a congestion event is detected

Reviewer: ```suggestion
During congestion avoidance, after a congestion event is detected
```
